pygmt/tests/test_solar.py

```diff
@@ -0,0 +1,131 @@
+"""
+Tests for solar.
+"""
+import datetime
+
+import pytest
+from pygmt import Figure
+from pygmt.exceptions import GMTInvalidInput
+
+
+@pytest.mark.mpl_image_compare
+def test_solar_default_terminator():
+    """
+    Test passing the solar argument with a time string and no terminator type
+    to confirm the default terminator type.
+    """
+    fig = Figure()
+    fig.solar(
+        region="d",
+        projection="W0/15c",
+        frame="a",
+        terminator_datetime="1990-02-17 04:25:00",
+    )
+    return fig
```

Comment:
Is the image `test_solar_default_terminator.png` the same as `test_solar_set_terminator_datetime.png`? If yes, we can change the baseline image by
```
@pytest.mark.mpl_image_compare(filename="test_solar_set_terminator_datetime.png")
```
so we can have one fewer image.

Reply:
@seisman Think I fixed it